src/eradius_counter.erl

```diff
@@ -217,50 +217,62 @@ server_stats(Func) ->
-do_inc_counter(replies,           Counters = #client_counter{replies = Value})           -> Counters#client_counter{replies = Value + 1};
-do_inc_counter(accessRequests,    Counters = #client_counter{accessRequests = Value})    -> Counters#client_counter{accessRequests = Value + 1};
-do_inc_counter(accountRequests,   Counters = #client_counter{accountRequests = Value})   -> Counters#client_counter{accountRequests = Value + 1};
-do_inc_counter(coaRequests,       Counters = #client_counter{coaRequests = Value})       -> Counters#client_counter{coaRequests = Value + 1};
-do_inc_counter(discRequests,      Counters = #client_counter{discRequests = Value})      -> Counters#client_counter{discRequests = Value + 1};
-do_inc_counter(retransmissions,   Counters = #client_counter{retransmissions = Value})   -> Counters#client_counter{retransmissions = Value + 1};
-do_inc_counter(timeouts,          Counters = #client_counter{timeouts = Value})          -> Counters#client_counter{timeouts = Value + 1};
-do_inc_counter(accessAccepts,     Counters = #client_counter{accessAccepts = Value})     -> Counters#client_counter{accessAccepts = Value + 1};
-do_inc_counter(accessRejects,     Counters = #client_counter{accessRejects = Value})     -> Counters#client_counter{accessRejects = Value + 1};
-do_inc_counter(accessChallenges,  Counters = #client_counter{accessChallenges = Value})  -> Counters#client_counter{accessChallenges = Value + 1};
-do_inc_counter(accountResponses,  Counters = #client_counter{accountResponses = Value})  -> Counters#client_counter{accountResponses = Value + 1};
-do_inc_counter(coaNaks,           Counters = #client_counter{coaNaks = Value})           -> Counters#client_counter{coaNaks = Value + 1};
-do_inc_counter(coaAcks,           Counters = #client_counter{coaAcks = Value})           -> Counters#client_counter{coaAcks = Value + 1};
-do_inc_counter(discNaks,          Counters = #client_counter{discNaks = Value})          -> Counters#client_counter{discNaks = Value + 1};
-do_inc_counter(discAcks,          Counters = #client_counter{discAcks = Value})          -> Counters#client_counter{discAcks = Value + 1};
-do_inc_counter(badAuthenticators, Counters = #client_counter{badAuthenticators = Value}) -> Counters#client_counter{badAuthenticators = Value + 1};
-do_inc_counter(packetsDropped,    Counters = #client_counter{packetsDropped = Value})    -> Counters#client_counter{packetsDropped = Value + 1};
-do_inc_counter(unknownTypes,      Counters = #client_counter{unknownTypes = Value})      -> Counters#client_counter{unknownTypes = Value + 1};
-do_inc_counter(pending,           Counters = #client_counter{pending = Value})           -> Counters#client_counter{pending = Value + 1}. 
+do_inc_counter(requests,               Counters = #nas_counter{requests = Value})                  -> Counters#nas_counter{requests = Value + 1};
```

Comment:
Only white space changes?

Reply:
Not only white space changes, was added also :
```
accountRequestsStart, accountRequestsStop, accountResponsesUpdate, accountResponsesStart, accountResponsesStop, accountRequestsUpdate %% For #nas_counter{}
% AND
accountRequestsStart, accountRequestsStop, accountResponsesUpdate, accountResponsesStart, accountResponsesStop, accountRequestsUpdate %% For #client_counter{}

```

Reply:
Could you please just add new clauses to the end, in other way diff touches all the lines and it is hard to review what really was changed